exchanges/btse/btse_types.go

```diff
@@ -2,9 +2,23 @@ package btse
 
 import "time"
 
+// OverviewData stores market overview data
+type OverviewData struct {
+	High24Hr         string `json:"high24hr"`
+	HighestBid       string `json:"highestbid"`
+	Last             string `json:"last"`
+	Low24Hr          string `json:"low24hr"`
+	LowestAsk        string `json:"lowest_ask"`
+	PercentageChange string `json:"percentage_change"`
```

Comment:
The field is `percent_change`. Please review all fields to be sure they match just in case

Reply:
This still needs a look

Reply:
This still still needs a look 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀 👀